apps/shipments/models/shipment.py

```diff
@@ -511,6 +523,24 @@ def drop_off(self, document_id=None, raw_asset_physical_id=None, **kwargs):
 
         self.delivery_act = datetime.now(timezone.utc)  # TODO: pull from action parameters?
 
+    def validate_gtx(self):
+        validation_status = GTXValidation.VALIDATION_FAILED
+        try:
+            response = requests.post(settings.GTX_VALIDATION_URL,
+                                     {'asset_physical_id': self.asset_physical_id, 'shipment_id': self.id})
+            if response.ok:
+                validation_status = GTXValidation.VALIDATED
+            else:
+                LOG.info(f'Invalid response from GTX, returned with status {response.status_code}')
+        except NewConnectionError:
```

Comment:
Is there a specific reason why you are catching this exception instead of `requests.exceptions.ConnectionError`?

Reply:
That was the type that I got when I turned off my internet to purposefully produce an error connecting. I can use the `ConnectionError` instead.